server/views/pages/checkTransfer.njk

```diff
@@ -1,75 +1,97 @@
 {% extends "../partials/layout.njk" %}
-{% set pageTitle = "Check transfer" %}
+{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
```

Comment:
meh.. haven't you been adding these to the layout template?

Reply:
yes, I have. 
I'll move it over in next ticket